### PR TITLE
feat: add crash reporting integration in demo app

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
Add [appcenter crash reporting library](https://docs.microsoft.com/en-us/appcenter/sdk/crashes/android) to the demo app to diagnosis crash reports in appcenter. 

## Links
- MINI-2009

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by minh-rakuten — APPROVED** [review]
Can't wait to see this works!

File: testapp/build.gradle [hunk 1]
```diff
@@ -51,6 +52,7 @@ android {
             projectId               : property("HOST_PROJECT_PROD_ID") ?: "test-host-project-id",
             subscriptionKey         : property("HOST_APP_PROD_SUBSCRIPTION_KEY") ?: "test-subs-key",
             adMobAppId              : property("PROD_ADMOB_APP_ID") ?: "",
+            appCenterSecret         : property("PROD_APPCENTER_SECRET") ?: "",
```
Comment (minh-rakuten) [inline testapp/build.gradle:55]: I think we only need `APPCENTER_SECRET` because there is only single AppCenter to manage both staging and production apk.

**Review by corycaywood — COMMENTED** [review]
Nice, it's looking good 👍 

But there's an issue that we won't be able to see code lines when the APK has been minified with R8. You have to upload the mappings to App Center: https://docs.microsoft.com/en-us/appcenter/diagnostics/android-deobfuscation#app-center-cli

You could add this to the App Center publish step on CircleCI ([see here](https://github.com/rakutentech/android-miniapp/blob/master/.circleci/config.yml#L40)):
```
post-steps:
  - run:
      command: appcenter crashes uploading-mappings // etc..
```

I haven't done this before, so I'm not sure how well it works 🤔  I guess we could just manually upload the mappings for now if it's too difficult to do automatically. What do you think?

**Comment by khairul-alam-licon** [comment]
@corycaywood Yeah, Thank you for pointing out the case when minified with R8. Automatically uploading the `mapping.txt` files would be nice but at this moment, it looks like a bit tricky to identify few configuration values e.g. `--mapping --version-name --version-code`. In that case, can we forward with manual uploading for now?

**Comment by corycaywood** [comment]
> @corycaywood Yeah, Thank you for pointing out the case when minified with R8. Automatically uploading the `mapping.txt` files would be nice but at this moment, it looks like a bit tricky to identify few configuration values e.g. `--mapping --version-name --version-code`. In that case, can we forward with manual uploading for now?

Yeah, I think it's okay for now to upload the mappings manually. Anyways, we only need the mappings in the case that we need to investigate a crash, so probably we can just manually upload them when we need to see the details of a crash.

**Review by corycaywood — APPROVED** [review]
LGTM 👍

**Comment by khairul-alam-licon** [comment]
Thank you for the reviews!

**Pull request merged**